agent/exploits/cve_2024_37383.py

```diff
@@ -0,0 +1,36 @@
+"""Agent Asteroid implementation for CVE-2024-37383"""
+
+import re
+
+from agent import definitions
+from agent import exploits_registry
+from agent.exploits import webexploit
+
+VULNERABILITY_TITLE = "XSS via SVG Animate Attributes in Roundcube Webmail"
+VULNERABILITY_REFERENCE = "CVE-2024-37383"
+VULNERABILITY_DESCRIPTION = (
+    "Roundcube Webmail before version 1.5.7 and versions 1.6.x before 1.6.7 contains a "
+    "cross-site scripting vulnerability that can be exploited via SVG animate attributes."
+)
+RISK_RATING = "HIGH"
```

Comment:
```suggestion
RISK_RATING = "MEDIUM"
```
Just a medium..
![image](https://github.com/user-attachments/assets/820a01f4-946b-451b-8211-6b6052c100ec)

Reply:
Ok thanks.